README.md

```diff
@@ -1,6 +1,9 @@
 # cidr-filtering-cni
 
-This plugin allows user to define allow list for ingress and egress packet filtering for a Kubernetes pod.
+This CNI plugin isolates pods from external endpoints. By default, all ingress and egress traffic is denied.
+Users may create special configMaps in the pod namespace to indicate allowed ingress or egress connections.
+This can be done in both directions - ingress and egress - for network layers L3 and L4.
```

Comment:
ingress / egress feels somewhat redundant - they appear twice in consecutive sentences.